sync.cmd

```diff
@@ -0,0 +1,25 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set _args=%*
+if /I [%1]==[/?] goto Usage
+if [%1]==[]  set _args=-p
+
+echo Running executor.cmd %_args%
+call executor.cmd %_args%
+if NOT [%ERRORLEVEL%]==[0] (
+  echo ERROR: An error occurred. There may have been networking problems so please try again in a few minutes.
+  exit /b 1
+)
+
+echo Done Syncing.
+exit /b 0
+
+:Usage
```

Comment:
Given there is only one option and it is also the default option why do we need Usage?